climada/util/plot.py

```diff
@@ -472,3 +472,70 @@ def get_transformation(crs_in):
     except KeyError:
         units = '°'
     return crs_epsg, units
+
+
+def bar_plot(ax, data, colors=None, total_width=0.8, single_width=1, legend=True):
+    """Draws a bar plot with multiple bars per data point.
+    https://stackoverflow.com/questions/14270391/python-matplotlib-multiple-bars
+
+    Parameters
+    ----------
+    ax : matplotlib.pyplot.axis
+        The axis we want to draw our plot on.
+
```

Comment:
Perhaps not so easy to get that one needs to define this like `ax = plt.subplot()` beforehand. 
How about merging this into a working example, and include a proper example section in the doc. Something like:

```
Example
--------
data = {
            "var1":[1,2,3],
            "var2":[1,2,3],
            "var3":[1,2,3],
        }

ax = plt.subplot()
bar_plot(ax, data)
```

I think it's useful to also provide in the example how to include x-axis labels then, as probably this would be a frequent operation and it's not 100% straightforward (e.g. `ax.set_xticklabels( ('one', 'two', 'three') )`  would add all those names only to the first group. one needs to explicitly set the positions now with set_xticklabels(), cf answer 2 in your stackoverflow link)

Reply:
I added the xticklabels as optional argument.

Reply:
Added xticklabels as optional parameter and completed the example.